### ISSUE TITLE
Could I input covariate file into BayesR model?

### ISSUE THREAD

**Issue by YinLiLin** [syntheke/bayesR#6]
Hi, 

I want to use BayesR model to do GWAS for my traits, and there are some covariates needed to be taken into consideration, but no descriptions about how to add covariates in the manual, so is there any way to input covariate file into BayesR software?

Thanks,
Best!

**Comment by syntheke** [comment]
There is no option to fit covariates. You could try to pre-adjust the phenotypes and use the residuals of the model as input for BayesR.

**Comment by leticia-lara** [comment]
Dear Luke,

I installed and compiled the bayesRv2 but when I try to use the flag -covar with a .txt file containing covariates in the column with space delimiter, I receive this message error: "Unknown command line option :-covar". 
Furthermore, I found no information about this flag in the tutorials "fast(er)BayesR.pdf" and "BayesRmanual-0.75.pdf". Which manual are you talking about?

Thank you.
